apps/condo/domains/notification/tasks/sendMessageBatch.helpers.js

```diff
@@ -1,72 +1,120 @@
-const { get, isString } = require('lodash')
+const { get, isString, isEmpty } = require('lodash')
 
 const conf = require('@open-condo/config')
 
 const { getLogger } = require('@open-condo/keystone/logging')
+const { md5 } = require('@condo/domains/common/utils/crypto')
 
 const { sendMessage } = require('@condo/domains/notification/utils/serverSchema')
 
 const {
     SMS_TRANSPORT, EMAIL_TRANSPORT, PUSH_TRANSPORT,
     CUSTOM_CONTENT_MESSAGE_TYPE,
+    CUSTOM_CONTENT_MESSAGE_PUSH_TYPE,
+    CUSTOM_CONTENT_MESSAGE_EMAIL_TYPE,
+    CUSTOM_CONTENT_MESSAGE_SMS_TYPE,
 } = require('../constants/constants')
 
 const EMAIL_FROM = 'noreply@doma.ai'
 const DATE_FORMAT = 'YYYY-MM-DD'
 const IS_EMAIL_REGEXP = /^\S+@\S+\.\S+$/
 const IS_PHONE_REGEXP = /^\+79\d{9}$/
```

Comment:
country dependant regex

Reply:
that's true, by now we don't send SMS to any other countries - it's too expensive.